2016-07/july-15.md

```diff
@@ -0,0 +1,57 @@
+
+#### Typescript Typings
+
+* Sean brought up that there has been multiple responses from individuals to not only bring typings to webpack 2.x, but also include official typings into the repository. Sean mentioned that the benefits could allow people to use typescript in their configurations and have automatic type validation. Tobias asked if we would have these tested incase changes are made. Sean said that we could write tests that allow the types to test against the webpack compiler and configuration and plugin api & that it would be a huge benefit for users and developers on core. A typings file would simply be added to the repository and a `'typings'` field added to the package.json. Everyone agreed to this as long as it is tested and integrated into the CI process. Sean mentioned he has one or two people already offering to assist him with this and will now look into this even further. 
+
+#### Documentation
+
+* Stubs for learning resources continue to pour into the github. 
+
+* Sean has offered to take on some of the Plugin API documentation when he gets around to it after his angular-cli webpack integration work. Also asked if Tobias would be willing to write the high level overview documentation for the Resolver instance. It is currently the least documented Plugin API feature and it would be great to have Tobias write it so that no stones are unturned about its use. 
+
+* The `antwar` [branch](https://github.com/webpack/webpack.io/tree/antwar) which contains most of Juho's work is nearing its point of completion. Juho states that a PR will be coming within the next week or so with all the content changes. 
+
+#### Slack
+
+* Sean recapped that he finally set up and sent out slack to the core team members. The team decided that we would try it out, as well as invite plugin and loader authors in hopes of specifically communicating with them more efficently. 
+
+#### Core Loaders and Plugins
+
+* Juho has created a document with initial sorting and assignment for loaders and plugins. Sean mentioned that he will go through this list and add anyone else that remains. Once this list is approved we will send to Tobias to extend invitations to the organization and repos.
+
+#### CSS Loader & CSS Modules 
+
+* Johannes brought up that there are still a number of performance related issues in regards to `css-loader` and inquired whether or not it would be possible to isolate the css-modules code from the loader. Tobias said that the changes he had made which were thought to increase performance were not as good as promised. Johannes asked if we could instead have two separate loaders so that people who do not want to use css-modules can simply use a faster lightweight css-loader option instead. Tobias said it may be possible and would look into it.
+
+### Takeaways
+* Finalize vetting people for core loaders and plugins (Sean)
```

Comment:
We should have the approval and First list of members agreed on in the next meeting.